### ISSUE TITLE
Lowercase italic k

### ISSUE THREAD

**Issue by mwsundberg** [indestructible-type/Bodoni#16]
In all font weights and optical sizes of the italics there is a slight issue with the lowercase _'k'_ character, the two lines of the intersect with an internal gap. In some styles (Fatface 11) it is less noticeable, yet particularly at larger optical sizes and in the variable font it stands out.
![image](https://user-images.githubusercontent.com/5696404/67403545-18ab2880-f580-11e9-89b3-c836aa9938cc.png)

**Comment by EwonRael** [comment]
hmmm...

This is an interesting design challenge because at the very large sizes I like having that space, and at the smaller sizes I like having it absent, but I also want it to be interpolatable.

I will think about this and see if I come up with something agreeable.

**Comment by mwsundberg** [comment]
Hmm, it is a bit nice at the higher optical sizes in the thinner weights. It would be amazing to have a stylistic alternative character for gapped or gapless, or somehow automatically switch versions for certain weight-size combos (like in the attached image, just my suggestion). I have no experience with font design or software, yet I know Adobe Illustrator, would it be helpful if I made svg versions of glyphs or something, if you'd like help? I don't just want to complain, if that makes sense.
![Weights for fixing](https://user-images.githubusercontent.com/5696404/67712061-9ebcda00-f999-11e9-8b9f-5ffd91245cac.png)

**Comment by EwonRael** [comment]
Yes, having an automatic switch between versions is an option worth considering as that is doable within variable fonts. 

I will play around with various solutions and come up with something for the next release. 

It wouldn't bother me if you just reported issues and didn't do anything more. I know more about the ins-and-outs than anyone else and I like doing this work.

That being said if you _did_ want to play around with the font files you could download [TruFont](https://github.com/trufont/trufont) and open some of the .ufo files up.

Thanks for the support!

**Comment by davelab6** [comment]
@EwonRael is this resolved? :)

**Comment by EwonRael** [comment]
Well, to be honest this is a little complicated. I've been playing with it and I've made some adjustments in the latest release I'm happy with, but I'm not convinced this is a solve issue. The newer version is better than the version we talked about here, but I think playing with it more might lead to even more agreeable solutions. So I guess better but not solved?
